app/views/steps/claim_type/edit.html.erb

```diff
@@ -1,25 +1,21 @@
 <% title t('.page_title') %>
-<% step_header %>
+<% step_header(path: applications_path) %>
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
     <%= govuk_error_summary(@form_object) %>
     <%= step_form @form_object do |f| %>
       <%= f.govuk_radio_buttons_fieldset :claim_type, legend: { tag: 'h1', size: 'xl' } do %>
-        <%= f.govuk_radio_button :claim_type, ClaimType::NON_STANDARD_MAGISTRATE.value do %>
+       <h3 class="govuk-heading-m"><%= 'What is you unique file number ?'  %></h3>
+       <%= f.govuk_text_field :ufn, width: 'one-third', autocomplete: 'off' %>
+        <h3 class="govuk-heading-m"><%= 'What are you claiming for ?'  %></h3>
```

Comment:
Should this read `'What are you claiming for?'` rather than `'What are you claiming for ?'`